war/src/main/webapp/help/project-config/downstream.html

```diff
@@ -6,4 +6,8 @@
   Other than the obvious use case where you'd like to build other projects that have a dependency
   on the current project, this can also be useful to split a long build process in to multiple
   stages (such as the build portion and the test portion).
+
+  <p>
+    Projects located inside folders can be specified using an absolute path starting with "/" or a
```

Comment:
Folders Plugin is not a part of the Jenkins core. If you reference it, there should be a hyperlink to the plugin at least.

It also worths to reference other similar project types (e.g. Multibranch)